app.env

```diff
@@ -0,0 +1,19 @@
+POSTGRES_HOST=127.0.0.1
+POSTGRES_PORT=6500
+POSTGRES_USER=admin
+POSTGRES_PASSWORD=password123
+POSTGRES_DB=tiny_site
+
+CLIENT_ORIGIN=http://localhost:3000
+
+TEST_DB_URL="postgresql://admin:password123@localhost:6500/tiny_site?sslmode=disable"
+
+
+DOMAIN="localhost"
```

Comment:
I suggest figure out about the hosted database and can we store these values in GitHub secrets please @vinit717